miden-tx/src/tests/mod.rs

```diff
@@ -377,6 +377,79 @@ fn executed_transaction_account_delta() {
     );
 }
 
+#[test]
+fn test_empty_delta_nonce_update() {
+    let tx_context = TransactionContextBuilder::with_standard_account(
+        ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN,
+        ONE,
+    )
+    .build();
+
+    let mut executor: TransactionExecutor<_, ()> =
+        TransactionExecutor::new(tx_context.clone(), None);
+    let account_id = tx_context.tx_inputs().account().id();
+    executor.load_account(account_id).unwrap();
+
+    let tx_script = format!(
+        "\
+        use.miden::account
+        use.miden::contracts::wallets::basic->wallet
+
+        ## ACCOUNT PROCEDURE WRAPPERS
+        ## ========================================================================================
+        #TODO: Move this into an account library
```

Comment:
What does this `TODO` mean here?

Reply:
I think the initial idea was to move account procedure wrappers to the library, but I'm not sure that it is valid for my test. I'll remove this line.